### PR TITLE
Css four column layout- David Gonzalez

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
image

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <h2 class="heading">How it Works</h2>
+      </div>
+      <div class="row">
+        <div class="columns"><img src="images/screen-1.png"> </div>
+        <div class="columns"><img src="images/screen-2.png"></div>
+        <div class="columns"><img src="images/screen-3.png"></div>
+        <div class="columns"><img src="images/screen-4.png"></div>
+      </div>
+      <div class="row">
+        <div class="columns-center">
```
Comment (Zhiwutian) [inline css-four-column-layout/index.html:21]: there doesn't appear to be a `columns-center` selector in the CSS, not sure what this is doing

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  background-color: #f7f8fa;
+  margin: auto;
+  padding-left: 7%;
+  padding-right: 10%;
+}
+
+h1 {
+  text-align: center;
+  font-family:
+    Barlow,
```
Comment (Zhiwutian) [inline css-four-column-layout/styles.css:12]: I would have these on the same line as the `font-family` property

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking good David, a couple small updates in the code, and one visual update. The text below the images looks a little off center, so make sure to adjust that.

**Comment by David-Gonzalez11** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**